fastlane/Fastfile

```diff
@@ -6,25 +6,112 @@
 #     https://docs.fastlane.tools/actions
 #
 
-# Uncomment the line if you want fastlane to automatically update itself
-# update_fastlane
+# There are a few env variables defined in the .env file in
+# this directory (fastlane/.env)
+
+
+# unlocks keychain if KEYCHAIN_PASSWORD variable is present
+# (to be used on CI machines)
+def unlock_keychain_if_needed
+    if ENV["KEYCHAIN_PASSWORD"]
```

Comment:
@mandrigin you might want to handle the case when env variable isn't setup, and send some meaningful message as `UI.error("...")`, since build will fail anyway with locked keychain.

I really like the way artsy treat open source contributors when they give suggestions on how to build app without being part of organization as part of the flow https://github.com/artsy/eidolon/blob/e90a1ffd38d73179a6fe2174587dcd98988efc48/fastlane/Fastfile#L37

Reply:
@nemoeslovo yeah, good point!
theoretically the keychain shouldn't be locked for non ssh builds, but it'd be nice to handle this case too

Reply:
they also support this magic value in their code for OSS contributors, right?